app/components/supplementary-file-browser.js

```diff
@@ -6,11 +6,21 @@ export default Ember.Component.extend({
     scrollAnim: '',
     numShowing: 6,
     selectedFile: {},
-
+    showArrows: Ember.computed('numShowing', 'supplementList.[]', function() {
+        return (this.get('numShowing') < this.get('files').length);
+    }),
     files: Ember.computed('fileList', 'fileList.[]', 'primaryFile', function() {
         //Returns the list with primaryFile moved to the front
         let files = this.get('fileList');
 
+        //TODO: Grab get primary file from preprint object
+        if (files) {
+            const fakePrimary = files.get('firstObject');
+            this.set('primaryFile', fakePrimary);
+            this.set('selectedFile', fakePrimary);
```

Comment:
You can set multiple properties at once 
http://emberjs.com/api/classes/Ember.Component.html#method_setProperties